docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.7"
+services:
+  curity:
+    container_name: idsvr
+    environment:
+      PASSWORD: Secr3t
+    image: curity/idsvr
+    ports:
+      - "6749:6749"
+      - "4466:4466"
```

Comment:
**expose** should be enough for this port.

Reply:
well... since the containers are within the same network we do not have to expose or publish any port for the systems to be able to communicate. However, people may want to investigate the system and understand what's going on under the hood. Don't you think?

Reply:
I would also add the 8443 port, so they can actually run some flows and see the results of those in the dashboard